_posts/slow-down.md

```diff
@@ -0,0 +1,69 @@
+
+## Slow is Smooth. Smooth is Fast
+
+With Udi, we discussed the "Slow is Smooth. Smooth is Fast" SEAL and Delta Force motto. The motto can be summarized as _you cannot go fast without first slowing down_. I quickly connected it to my swimming activities.
+
+Swimming is an extraordinarily complex and challenging activity. Swimming fast comes with many subtle challenges. Technique is everything; it's much more important than strength. And probably the most difficult part is breathing.
+
+When swimming front crawl or butterfly for the sake of the sample, the breath pace is determined by strokes pace, and swimmers cannot breath when their body needs or at the rhythm they want. Usually, what happens to amateur swimmers when they start is that they try to swim faster by stroking faster. Stroking faster comes with breathing at a faster pace and with less time for each breath. Less time to breathe means less oxygen exchange which causes hypoxia that eventually leads to less effective muscles. The result is that they go slower instead of faster.
+
+Why is that so complex? Let me use the front crawl stroke as an example. The front crawl stroke is composed of at least five stages:
+
+- The initial part of the stroke is how and where hands and arms get into the water.
+- Then there is the catch when swimmers start pulling water with their hands and forearms. How the elbow is positioned is crucial to a good catch.
+- After catching water, the next stage is pulling. In this case, a critical factor is the body angle needed to distribute strength between pectorals and back muscles properly.
+- Then, how the stroke is finished is the next step, needed to squeeze out all possible energy from the previous pulling stage.
+- Finally, getting elbows out of the water with hands nearly touching ears stretching over the surface.
+
+Add to the complex mentioned stages the need to breathe and coordinate with leg movements crucial to strokes and breathing. Even if you never practiced front crawl, you probably got a good enough picture of the complexity.
+
+Swimmers, especially beginners, need to slow down to concentrate on breathing and many other essential details. Only by slowing down can they focus on each step in isolation, on the required technique, and repeat the movements repeatedly until muscle memory kicks in. They become a natural part of the swimming style. At which point going faster becomes a secondary effect. Until you start focusing on legs, but that's another complex story.
+
+## How can we swim the ocean of thousands of activities?
+
+Can we learn something from swimmers, and many other conceptually similar activities, to improve our efficiency? For example, if we believe we're achieving little to nothing, how can we go "faster"? What would a swimmer do?
```

Comment:
Not sure if it's worth mentioning but practicing a difficult piano piece is the same thing. You start slow to work out the notes and fingering and dynamics until muscle memory kicks in, then you speed it up.